compiler+runtime/src/cpp/jank/read/parse.cpp

```diff
@@ -1166,6 +1168,15 @@ namespace jank::read::parse
                                token };
   }
 
+  processor::object_result processor::parse_ratio() {
+    auto const token(token_current->expect_ok());
+    ++token_current;
+    const lex::ratio ratio_data = boost::get<lex::ratio>(token.data);
```

Comment:
Please use `auto const &ratio_data(boost::get<lex::ratio>(token.data));`.

1. Uses direct initialization
2. Uses a ref to const, to avoid copying
3. Uses auto, since the type is clear from the right hand side